superset-frontend/src/types/Chart.ts

```diff
@@ -44,4 +45,7 @@ export type Slice = {
   slice_name: string;
   description: string | null;
   cache_timeout: number | null;
+  url?: string;
 };
+
+export default Chart;
```

Comment:
Yikes, i don't even want to know why we have a `Slice` and a `Chart` type...

Reply:
I think one has stricter typing following the backend API response; another is for temporary chart state on the client side (we should probably consolidate these two).